src/core/client/auth/test/create.tsx

```diff
@@ -45,6 +46,8 @@ export default function create(params: CreateParams) {
     postMessage: new PostMessageService(),
     browserInfo: { ios: false },
     uuidGenerator: createUUIDGenerator(),
+    eventEmitter: new EventEmitter2(),
```

Comment:
Just note that the default event emitter has caps on the number of listeners.

Reply:
Interesting, yeah not directly using this yet, but we will eventually.